src/Screens/components/colors.ts

```diff
@@ -6,6 +6,7 @@ const colors = {
   white: '#FFF',
   gray: '#8F9EAC',
   black: '#000',
+  transparentBlack: '#0002',
```

Comment:
Cool, I did not know that you can do transparency like this. AFAIK there is also some opacity or something similar in style props.